resources/codegen_inputs/svg.rs

```diff
@@ -0,0 +1,39 @@
+#![parse_module(read_fonts::tables::svg)]
+
+/// [SVG](https://learn.microsoft.com/en-us/typography/opentype/spec/svg)
+#[tag = "SVG "]
+table SVG {
```

Comment:
We’ve discussed naming table types with the same capitalization as the tag but haven’t made that change yet so this should be `Svg` to match the current naming convention. The SVG in the other codegen’d types is fine. 

Also, just for consistency, the doc comment here should match the one on the module, e.g. “The SVG table” with the link on SVG.